# This file is distributed under the same license as the Django package.
#
# Translators:
# Jan Munclinger <jan.munclinger@gmail.com>, 2013
# Jannis Leidel <jannis@leidel.info>, 2011
# Tomáš Ehrlich <tomas.ehrlich@gmail.com>, 2015
# Vláďa Macek <macek@sandbox.cz>, 2013-2014
# Vláďa Macek <macek@sandbox.cz>, 2015-2017,2019
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-07-25 00:15+0300\n"
"PO-Revision-Date: 2019-09-19 08:58+0000\n"
"Last-Translator: Vláďa Macek <macek@sandbox.cz>\n"
"Language-Team: Czech (http://www.transifex.com/django/django/language/cs/)\n"
"Language: cs\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=4; plural=(n == 1 && n % 1 == 0) ? 0 : (n >= 2 && n "
"<= 4 && n % 1 == 0) ? 1: (n % 1 != 0 ) ? 2 : 3;\n"

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:46
msgid "Personal info"
msgstr "Osobní údaje"

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:47
msgid "Permissions"
msgstr "Oprávnění"

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:50
msgid "Important dates"
msgstr "Důležitá data"

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:134
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr "Položka \"%(name)s\" s primárním klíčem \"%(key)r\" neexistuje."

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:144
msgid "Password changed successfully."
msgstr "Změna hesla byla úspěšná."

#: env/lib/python3.8/site-packages/django/contrib/auth/admin.py:164
#, python-format
msgid "Change password: %s"
msgstr "Heslo pro uživatele %s: změnit"

#: env/lib/python3.8/site-packages/django/contrib/auth/apps.py:15
msgid "Authentication and Authorization"
msgstr "Autentizace a autorizace"

#: env/lib/python3.8/site-packages/django/contrib/auth/base_user.py:48
msgid "password"
msgstr "heslo"

#: env/lib/python3.8/site-packages/django/contrib/auth/base_user.py:49
msgid "last login"
msgstr "poslední přihlášení"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:40
msgid "No password set."
msgstr "Heslo nenastaveno."

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:45
msgid "Invalid password format or unknown hashing algorithm."
msgstr "Neplatný formát hesla nebo neplatný hashovací algoritmus."

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:87
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:334
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:407
msgid "The two password fields didn’t match."
msgstr "Hesla se neshodují."

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:90
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:143
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:179
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:411
msgid "Password"
msgstr "Heslo"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:96
msgid "Password confirmation"
msgstr "Potvrzení hesla"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:99
#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:420
msgid "Enter the same password as before, for verification."
msgstr "Zadejte pro ověření stejné heslo jako předtím."

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:145
msgid ""
"Raw passwords are not stored, so there is no way to see this user’s "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""
"Hesla se neukládají přímo a tak je nelze zobrazit. Je ale možné je změnit "
"pomocí <a href=\"{}\">tohoto formuláře</a>."

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:186
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""
"Zadejte správnou hodnotu pole %(username)s a heslo. Pozor, obě pole mohou "
"rozlišovat malá a velká písmena."

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:189
msgid "This account is inactive."
msgstr "Tento účet je neaktivní."

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:252
msgid "Email"
msgstr "E-mail"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:337
msgid "New password"
msgstr "Nové heslo"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:343
msgid "New password confirmation"
msgstr "Potvrzení nového hesla"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:379
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr "Vaše současné heslo nebylo zadáno správně. Zkuste to znovu."

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:382
msgid "Old password"
msgstr "Současné heslo"

#: env/lib/python3.8/site-packages/django/contrib/auth/forms.py:417
msgid "Password (again)"
msgstr "Heslo (znovu)"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:259
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:333
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:429
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:489
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:520
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:556
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:592
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:630
msgid "algorithm"
msgstr "algoritmus"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:260
msgid "iterations"
msgstr "iterace"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:261
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:339
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:431
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:490
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:521
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:631
msgid "salt"
msgstr "hodnota salt"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:262
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:340
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:491
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:522
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:557
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:593
#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:632
msgid "hash"
msgstr "hash"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:334
msgid "variety"
msgstr "varieta"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:335
msgid "version"
msgstr "verze"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:336
msgid "memory cost"
msgstr "spotřeba paměti"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:337
msgid "time cost"
msgstr "doba"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:338
msgid "parallelism"
msgstr "paralelismus"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:430
msgid "work factor"
msgstr "faktor práce"

#: env/lib/python3.8/site-packages/django/contrib/auth/hashers.py:432
msgid "checksum"
msgstr "kontrolní součet"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:56
#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:108
msgid "name"
msgstr "název"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:60
msgid "content type"
msgstr "typ obsahu"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:62
msgid "codename"
msgstr "kódový název"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:67
msgid "permission"
msgstr "oprávnění"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:68
#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:111
msgid "permissions"
msgstr "oprávnění"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:118
msgid "group"
msgstr "skupina"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:119
#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:242
msgid "groups"
msgstr "skupiny"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:233
msgid "superuser status"
msgstr "superuživatel"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:236
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr ""
"Určuje, že uživatel má veškerá oprávnění bez jejich explicitního přiřazení."

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:245
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""
"Skupiny, do kterých tento uživatel patří. Uživatel dostane všechna oprávnění "
"udělená každé z jeho skupin."

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:253
msgid "user permissions"
msgstr "uživatelská oprávnění"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:255
msgid "Specific permissions for this user."
msgstr "Konkrétní oprávnění tohoto uživatele."

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:326
msgid "username"
msgstr "uživatelské jméno"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:329
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""
"Požadováno. 150 znaků nebo méně. Pouze písmena, číslice a znaky @/./+/-/_."

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:332
msgid "A user with that username already exists."
msgstr "Uživatel s tímto jménem již existuje."

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:335
msgid "first name"
msgstr "křestní jméno"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:336
msgid "last name"
msgstr "příjmení"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:337
msgid "email address"
msgstr "e-mailová adresa"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:339
msgid "staff status"
msgstr "administrační přístup"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:341
msgid "Designates whether the user can log into this admin site."
msgstr "Určuje, zda se uživatel může přihlásit do správy tohoto webu."

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:344
msgid "active"
msgstr "aktivní"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:347
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""
"Určuje, zda bude uživatel považován za aktivního. Použijte tuto možnost "
"místo odstranění účtů."

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:351
msgid "date joined"
msgstr "datum registrace"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:360
msgid "user"
msgstr "uživatel"

#: env/lib/python3.8/site-packages/django/contrib/auth/models.py:361
msgid "users"
msgstr "uživatelé"

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:102
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] "Heslo je příliš krátké. Musí mít délku aspoň %(min_length)d znak."
msgstr[1] "Heslo je příliš krátké. Musí mít délku aspoň %(min_length)d znaky."
msgstr[2] "Heslo je příliš krátké. Musí mít délku aspoň %(min_length)d znaků."
msgstr[3] "Heslo je příliš krátké. Musí mít délku aspoň %(min_length)d znaků."

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:112
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] "Heslo musí mít délku aspoň %(min_length)d znak."
msgstr[1] "Heslo musí mít délku aspoň %(min_length)d znaky."
msgstr[2] "Heslo musí mít délku aspoň %(min_length)d znaků."
msgstr[3] "Heslo musí mít délku aspoň %(min_length)d znaků."

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:151
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr "Heslo je příliš podobné obsahu pole %(verbose_name)s."

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:157
msgid "Your password can’t be too similar to your other personal information."
msgstr "Heslo nemůže být příliš podobné jinému údaji ve vašem účtu."

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:183
msgid "This password is too common."
msgstr "Heslo je příliš běžné."

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:188
msgid "Your password can’t be a commonly used password."
msgstr "Vaše heslo nemůže být takové, které je často používané."

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:198
msgid "This password is entirely numeric."
msgstr "Heslo se skládá pouze z čísel."

#: env/lib/python3.8/site-packages/django/contrib/auth/password_validation.py:203
msgid "Your password can’t be entirely numeric."
msgstr "Vaše heslo nemůže být čistě číselné."

#: env/lib/python3.8/site-packages/django/contrib/auth/templates/registration/password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr "Obnovení hesla na webu %(site_name)s"

#: env/lib/python3.8/site-packages/django/contrib/auth/validators.py:12
msgid ""
"Enter a valid username. This value may contain only English letters, "
"numbers, and @/./+/-/_ characters."
msgstr ""
"Zadejte platné uživatelské jméno. Hodnota může obsahovat pouze písmena bez "
"diakritiky, tj. háčků a čárek, číslice a znaky @/./+/-/_."

#: env/lib/python3.8/site-packages/django/contrib/auth/validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""
"Zadejte platné uživatelské jméno. Hodnota může obsahovat pouze písmena, "
"číslice a znaky @/./+/-/_."

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:160
msgid "Logged out"
msgstr "Odhlášeno"

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:217
msgid "Password reset"
msgstr "Obnovení hesla"

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:244
msgid "Password reset sent"
msgstr "Zpráva s obnovením hesla byla odeslána"

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:254
msgid "Enter new password"
msgstr "Zadejte nové heslo"

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:314
msgid "Password reset unsuccessful"
msgstr "Obnovení hesla bylo neúspěšné"

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:322
msgid "Password reset complete"
msgstr "Heslo bylo obnoveno"

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:334
msgid "Password change"
msgstr "Změna hesla"

#: env/lib/python3.8/site-packages/django/contrib/auth/views.py:357
msgid "Password change successful"
msgstr "Změna hesla byla úspěšná"
